src/main/resources/assets/templates/letters/chips/CSIDVDEFLET_content_v1.html

```diff
@@ -0,0 +1,66 @@
+<p class="subject-line">ACTION REQUIRED: Overdue Statutory Filings</p>
+<p class="subject-line"><span class="subject-line">[[${company_name}]] ([[${company_number}]])</span></p>
+<hr/>
+
+<p>
+    We are writing to inform you that [[${company_name}]] has failed to deliver the following documents to Companies House:
+    <ol>
+        <li>Annual confirmation statement (CS01 form) for the review period between <span id="cs-review-period-start">[[${cs_review_period_start}]]</span> and <span id="cs-review-period-end">[[${cs_review_period_end}]]</span>.</li>
+        <li>Director identity verification statements for all registered company directors.</li>
+    </ol>
+</p>
+
+<p>These statements must be filed together to comply with requirements under the Economic Crime and Corporate Transparency Act 2023, which introduced mandatory identity verification for all company directors and persons with significant control (PSCs).</p>
+
+<p>You must deliver the outstanding statements or, if you are unable to comply due to exceptional circumstances, submit the reason in writing by <span id="action-due-date">[[${action_due_date}]]</span>.</p>
+
+<p>Failure to file a confirmation statement constitutes an offence under Section 853L of the Companies Act 2006. In addition, any director who continues to act without verifying their identity will be committing an offence under Section 167M of the Companies Act 2006.</p>
+
+<p>
+    Where these requirements are not met, we may initiate enforcement action without further notice. This may include:
+    <ul>
+        <li>Prosecution of the company and its officers</li>
+        <li>Financial penalties imposed on individual directors and the company</li>
+        <li>Strike-off proceedings, resulting in the company being removed from the register</li>
+        <li>Disqualification of directors, preventing them from acting in similar roles in the future</li>
+    </ul>
+</p>
+
+<p class="emphasis">Instructions on how to comply with identity verification requirements and submit reasons for exceptional circumstances are on the back of this letter.</p>
+
+<p class="subject-line" style="page-break-before: always">There are 2 steps to comply with identity verification requirements:</p>
+<p class="subject-line-spacer"></p>
+
+<table>
+    <tr>
+        <td class="verification-requirements">
```

Comment:
This is using an HTML table for styling which is a bad practice. However we are only following the DEEP lead and reusing their styling so not a blocker